lib/helpers.js

```diff
@@ -31,9 +31,10 @@ export function getPackageRoot() {
 }
 
 function getAtomAppName() {
-  const match = atom.getVersion().match(/beta|nightly/);
+  const match = atom.getVersion().match(/-([a-z]+)(\d+|-)/);
```

Comment:
No chance we'll use uppercase letters in our channel names?  :smile:

Reply:
I had it there at first but took it out because I figured it wouldn't happen.  I can add it back!